mjc-spring-project/dao/src/main/java/com/epam/esm/dao/OrderRepository.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.esm.dao;
+
+import com.epam.esm.dao.entity.Order;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+/**
+ * The interface Order repository.
+ */
+@Repository
+public interface OrderRepository extends JpaRepository<Order, Long> {
+
+    /**
+     * Find by user id.
+     *
+     * @param userId   the user id
+     * @param pageable the pageable
+     * @return the page
+     */
+    @Query("select o from Order o where o.user.id = :userId and o.isDeleted = false")
```

Comment:
suggest to read point 4 from this article https://www.baeldung.com/hibernate-dynamic-mapping to not include isDeleted = false in each request

Reply:
fixed